### PR TITLE
Races document spell check

### PR DESCRIPTION
**Pull request by bleehu**
I also moved Detachment A Security Drone's Robotic Type to a Racial Trait.

### CODE REVIEW

File: CharacterCreation/01_Races.txt [hunk 1]
```diff
@@ -21,7 +21,7 @@ None
 
 //============================================================
 Tundari
-Society: Tribal system, lead by a counsil of elders.
+Society: Tribal system, lead by a counsel of elders.
```
Comment (Turtlelord26) [inline CharacterCreation/01_Races.txt]: Counsel is a verb. You wanted "Council".

File: CharacterCreation/01_Races.txt [hunk 2]
```diff
@@ -32,26 +32,26 @@ more temperate, jungle climate. 90% of the annual rainfall on the planet
 occurs on this jungly southern continent.
 
 Native Species: Tundarin
-This Feline-esque especies is covered in a soft, short fur ranging in many 
-colors, from browns, to oranges, blacks, greys, etc. And a multitude of paterns
+This Feline-esque species is covered in a soft, short fur ranging in many 
+colors, from browns, to oranges, blacks, grays, etc. And a multitude of pasterns
```
Comment (Turtlelord26) [inline CharacterCreation/01_Races.txt]: Pasterns -> Patterns.

File: CharacterCreation/01_Races.txt [hunk 3]
```diff
@@ -32,26 +32,26 @@ more temperate, jungle climate. 90% of the annual rainfall on the planet
 occurs on this jungly southern continent.
```
Comment (Turtlelord26) [inline CharacterCreation/01_Races.txt:28]: Line 28: "Upright" axis does not mesh with further data that a pole points toward the suns. Upright sounds like an axis perpendicular to the plane of the ecliptic. Also I'm not sure if a planet with that kind of tidal locking and axial direction is astronomically plausible.
Line 29: "it's" -> "its".
Line 29: Capitalize "The north pole points..."

File: CharacterCreation/01_Races.txt [hunk 4]
```diff
@@ -226,7 +227,7 @@ Seadevil
 -1 to base CHA, -1 to base FOR.
```
Comment (Turtlelord26) [inline CharacterCreation/01_Races.txt]: Line 220: FOR -> FORT

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]

**Comment by bleehu** [comment]
Made fixes suggested by Turtle, with one exception. I'm not going to mess with the astronomy around upright/tilted axes in the spell check merge. One problem at a time.

**Review by Turtlelord26 — APPROVED** [review]

File: CharacterCreation/01_Races.txt [hunk 5]
```diff
@@ -67,41 +67,42 @@ Stat Modifiers: Flexible - +1 DEX, +1 to PER
...
-reactor. Not affected by mind-influence, succeptable to hack attempts.
+Type: Cybernetic
 Size: Medium+
 Speed: Normal
 Starting Stat Points: 40
 Stat Modifiers: INT + 1, FORT + 2. LUCK -3.
 Language: Basic
 
 Racial Traits:
+Robotic. Does not need to eat, breath, or sleep. Must spend 6 hours to
```
Comment (1sourcecontrol) [inline CharacterCreation/01_Races.txt]: I know it's a small thing.. Can you change "Robotic." to "Robotic:" ?

File: CharacterCreation/01_Races.txt (hunk 4 above)
Comment (1sourcecontrol) [inline CharacterCreation/01_Races.txt]: I second Turtlelord here, can you chance "FOR" to "FORT" for consistencies sake?

File: CharacterCreation/01_Races.txt [hunk 6]
```diff
@@ -226,7 +227,7 @@ Seadevil
 -1 to base CHA, -1 to base FOR.
 +1 to DEX
 Low-light vision.
-Ability: Bioluminesce - Once per day.[Is an available skill] Persuade a being, sentient or not, that you are trustworthy. Victim must roll a will-save 50+(Siren points). Effect is dissipated when victims eyes are covered or lose sight of you. This takes a tremendous amount of effort and you take a -10 penalty if tired or bloodied. This ability will temporarily scramble a robot’s visual cortex, but they can adjust rapidly.
+Ability: Bioluminescence - Once per day.[Is an available skill] Persuade a being, sentient or not, that you are trustworthy. Victim must roll a will-save 50+(Siren points). Effect is dissipated when victims eyes are covered or lose sight of you. This takes a tremendous amount of effort and you take a -10 penalty if tired or bloodied. This ability will temporarily scramble a robot’s visual cortex, but they can adjust rapidly.
 
 Chondrich
 -1 to base CHA, +1 to both STR and FOR
```
Comment (1sourcecontrol) [inline CharacterCreation/01_Races.txt]: also here, for the "FOR" -> "FORT" change.

**Review by 1sourcecontrol — APPROVED** [review]
If you are fine with my nitpicks, go ahead and shipit after changing.

File: CharacterCreation/01_Races.txt (hunk 6 above)
Comment (bleehu) [reply CharacterCreation/01_Races.txt]: Can Do, Will Do! :)

File: CharacterCreation/01_Races.txt (hunk 4 above)
Comment (bleehu) [reply CharacterCreation/01_Races.txt]: I think I'll change, not chance ;) Will do!

Comment (1sourcecontrol) [reply CharacterCreation/01_Races.txt]: toupee

File: CharacterCreation/01_Races.txt (hunk 5 above)
Comment (bleehu) [reply CharacterCreation/01_Races.txt]: Can do, will do :)

**Comment by bleehu** [comment]
fixed nitpicks and ship't!

**Pull request merged**